keras_nlp/network_tests/test_load_ckpts.py

```diff
@@ -44,6 +44,26 @@ def test_load_bert(self, bert_variant, weights):
         }
         model(input_data)
 
+    def test_load_bert_backbone_string(self):
```

Comment:
we have a dict of keys we could traverse for all of these tests. should we just do this approach everywhere rather than a parameterized test? seems much more maintainable